conduct/code-of-conduct.md

```diff
@@ -40,11 +40,21 @@ except as necessary to protect vulnerable people from intentional abuse.
 
 ### Harassment in online channels
 
-Participants at `<EVENT>` may also participate in online chats, discussion forums, and mailing lists (including platforms such as Slack and Twitter). Please use these guidelines where engaging with particpants:
+Participants at `<EVENT>` may also participate online (see below). Please use these guidelines where engaging with particpants:
 
 * Avoid using overtly sexual or offensive usernames or profile photos which might detract from a friendly, safe and welcoming environment for all.
 * Do not publish text/screenshots of anything shared in a private communication channels without explicit consent from the author.  This includes screenshots of private messages to public channels, as well as conversations on public channels to anywhere outside of `<EVENT>` Slack.
 
+This CoC applies to the all `<EVENT>` online spaces, which includes, but is not limited to:
+
+* Twitter
+* Slack
+* Facebook
+* WhatsApp
+* Gitter
+
+This CoC does not apply only to online channels __during__ the conference. For example, if someone is harassed in a conference related online channel leading up to the event while discussing ideas for the event, it would be treated as a CoC violation.
```

Comment:
This seems like the wrong place to emphasise that because if someone was harassed at a pre-conference in-person event, the CoC would also apply.

We may want to mention that the CoC generally applies to the conference and pre-conference events. I don't think it's feasible to require that it apply after the conference, since an event will have wrapped up and the CoC committee disbursed (and most disciplinary options are no longer available)

Reply:
Okay, i'll change to during and pre-conf. And remove the example